todo_rxbloc_app/lib/feature_add_todo/bloc/add_todo_bloc_extensions.dart

```diff
@@ -0,0 +1,25 @@
+part of 'add_todo_bloc.dart';
+
+extension _AddTodoExtension on Stream {
+  Stream<NewTodoEnum> createTodo(
+    TodoService todoService,
+    Stream<String> setTitle,
+    Stream<String> setDescription,
+    Uuid uuid,
+    NavigationBlocType navigationBloc,
+  ) =>
+      withLatestFrom2<String, String, List<String>>(setTitle, setDescription,
+              (_, String title, String desc) => [title, desc])
+          .switchMap((value) async* {
+        if (value[0].isNotEmpty) {
```

Comment:
The list approach is kind of risky. I would recommend implementing a class that contain the title and description